Remove hardcoded CMAKE_CUDA_ARCHITECTURES so that flag is picked up instead.
author: micketeer@gmail.com
--- CMakeLists.txt.orig	2026-10-27 20:37:22.655141137 +0100
+++ CMakeLists.txt	2026-10-27 20:37:30.997240746 +0100
@@ -43,7 +43,6 @@
                 enable_language(CUDA)
                 message(STATUS "CUDA compiler detected at: ${CMAKE_CUDA_COMPILER}")
                 # Optionally allow the user to set CUDA architectures manually, otherwise set a default
-                set(CMAKE_CUDA_ARCHITECTURES 70 75 80 86)
                 message(STATUS "Using CUDA architectures: ${CMAKE_CUDA_ARCHITECTURES}")
                 set(BUILD_GPU_TARGET ON)
         else()
